easybuild/easyconfigs/g/GffCompare/GffCompare-0.10.6-foss-2017b.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'GffCompare'
+version = '0.10.6'
+
+homepage = 'https://github.com/gpertea/%(namelower)s'
+description = """GffCompare provides classification and reference annotation mapping and
+ matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://github.com/gpertea/%(namelower)s/archive/',
+    'https://github.com/gpertea/gclib/archive/',
+]
+sources = [
+    'https://github.com/gpertea/%(namelower)s/archive/v%(version)s.tar.gz',
+    'https://github.com/gpertea/gclib/archive/v0.10.3.tar.gz',
+]
```

Comment:
@HPC-UniOldenburg Please remove the full URL and leave only the file name in the `sources` section. Also prefer `%(version)s` to the hardcoded version in the names.